bindata/v4.1.0/config/defaultconfig.yaml

```diff
@@ -14,6 +14,51 @@ aggregatorConfig:
     certFile: /etc/kubernetes/static-pod-certs/secrets/aggregator-client/tls.crt
     keyFile: /etc/kubernetes/static-pod-certs/secrets/aggregator-client/tls.key
 apiServerArguments:
+  enable-admission-plugins:
```

Comment:
as discussed, would like to see some test that fails when this list diverges from upstream+static-diff. Some simple origin carry patch should be able to do that. Or I believe you can even call the kube-apiserver binary. The command line help text contains the list.

Reply:
> as discussed, would like to see some test that fails when this list diverges from upstream+static-diff. Some simple origin carry patch should be able to do that. Or I believe you can even call the kube-apiserver binary. The command line help text contains the list.

This is no worse than today with regard to in and out.  In fact, it has better failure modes on registration failures.  I suggest dealing with the e2e test separately.